CMakeLists.txt

```diff
@@ -280,7 +280,7 @@ set(HAVE_LIBROCKSDB 1)
 find_package(ZLIB REQUIRED)
 
 #option for LTTng
-option(WITH_LTTNG "LTTng tracing is enabled" ON)
+option(WITH_LTTNG "LTTng tracing is enabled" OFF)
```

Comment:
could you add a comment here? something like `# currently OFF by default, because lttng_gen_tp runs unconditionally and forces a rebuild`